complaints/views.py

```diff
@@ -173,7 +203,7 @@ def email_on_request(request_id, category, details, issue, user_email):
 
 
 def email_resolve(
-        request_id, request_status, issue, user_email, category, details, remark_user
+    request_id, request_status, issue, user_email, category, details, remark_user
 ):
     """Function to send email to the user when his request or compliant gets resolved"""
     subject = f"{issue} request_status"
```

Comment:
put {request_status} instead of request_status

Reply:
done